### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by larry0x**
This repo currently contains copies of mars-osmosis, mars-outpost, and mars-testing crates. Now that we have open sourced red-bank repo, we don't need these copies anymore.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -19,26 +14,28 @@ documentation = "https://docs.marsprotocol.io/"
...
-mars-testing    = { version = "0.1.0", path = "./packages/testing" }
-osmosis-std     = "0.13.2"
-schemars        = "0.8"
-serde           = { version = "1.0", default-features = false, features = ["derive"] }
-thiserror       = "1.0"
+cosmwasm-schema     = "1.2"
+cosmwasm-std        = "1.2"
+cw2                 = "1.0"
+cw-storage-plus     = "1.0"
+cw-utils            = "1.0"
```
Comment (grod220) [inline Cargo.toml:21]: All deps should use fully qualified paths

Comment (larry0x) [reply Cargo.toml]: @grod220 fixed

**Review by grod220 — APPROVED** [review]

File: Cargo.toml [hunk 2]
```diff
@@ -19,26 +14,28 @@ documentation = "https://docs.marsprotocol.io/"
...
-osmosis-std     = "0.13.2"
-schemars        = "0.8"
-serde           = { version = "1.0", default-features = false, features = ["derive"] }
-thiserror       = "1.0"
+cosmwasm-schema     = "1.2.1"
+cosmwasm-std        = "1.2.1"
+cw2                 = "1.0.1"
+cw-storage-plus     = "1.0.1"
+cw-utils            = "1.0.1"
+# FIXME: update these once they're published to crates.io
```
Comment (piobab) [inline Cargo.toml:22]: TBH I am not sure if we should push all packages to crates.io. I would prefer only `mars-red-bank-types`. The rest are more internal and used between contracts.

**Review by piobab — APPROVED** [review]

File: Cargo.toml (hunk 2 above)
Comment (larry0x) [reply Cargo.toml:22]: I think all packages should be published. Firstly there's no downside in doing so. Secondly it prevents someone else from creating a crate of the same name. Thirdly if there is any chance other devs want to import the contracts into their projects, it makes it easier.

**Pull request merged**